Removed Module name replacement from URL path

Resteklicken opened the pull request:
Closes #157

eznavy reviewed (commented):
URL looks much cleaner now 👍 I noticed the following: When clicking on an exercise, the module title shortly appears in the URL and gets replaced with the ID again.

Resteklicken commented:
> URL looks much cleaner now +1 I noticed the following: When clicking on an exercise, the module title shortly appears in the URL and gets replaced with the ID again.

Thanks for having a look!
Fixed two other instances of the module name being put into the title but there's still this little glimpse of 'undefined' when clicking on an exercise. I refactored a bunch of stuff there on #181 so maybe that will fix it ;)

hwgn reviewed (approved):
LGTM - not sure why the undefined string even shows up. The goToEditor function may not need the more detailed path but it's fine to do, too: 
When using `router.push('some-path-here')` without a slash `/` at the start, the route will be appended instead of replaced, like this:
`/current/path` - > `/current/path/some-path-here`
while if the string was `/some-path-here`, it behaves like this:
`/current/path` -> `/some-path-here`

hope that clears up some confusion :)
